doc/source/graph/svcorch.md

```diff
@@ -166,4 +158,18 @@ Note no metrics or extra data will be added to the request so this would need to
 
 ## Routing in Metadata
 
-The current default orchestrator in Go the "executor" does not return routing meta data in request calls. This is a [known issue](https://github.com/SeldonIO/seldon-core/issues/1823). 
\ No newline at end of file
+The current default orchestrator in Go the "executor" does not return routing meta data in request calls. This is a [known issue](https://github.com/SeldonIO/seldon-core/issues/1823). 
```

Comment:
This was re-added recently but requires environmental variable to be set. Worth noting it here.

Reply:
This is the PR that added it again: https://github.com/SeldonIO/seldon-core/pull/2723

Reply:
Just don't remember what was the environmental variable to set...

Reply:
Good spot @RafalSkolasinski ! Should I just remove that line?

Reply:
I would definitely make sure that this is documented somewhere around the orchestrator in a visible place as it was one of the blockers for ppl from moving to `executor`.

Reply:
If it's no longer an issue, wouldn't it be better to remove it completely? Thinking of 2 or 3 versions ahead, wouldn't it be confusing for users to see a specific section for something that works?

Or is it the case that users require some guidance to use the routing metadata?

Reply:
I've now added a note on the routing metadata injection env var @RafalSkolasinski. Would be great to hear your thoughts!

Reply:
It looks perfect, nice one :+1: